packages/assets-controllers/src/assetsUtil.test.ts

```diff
@@ -436,4 +439,134 @@ describe('assetsUtil', () => {
       expect(timestampsIncreasing).toBe(true);
     });
   });
+
+  describe('fetchAndMapExchangeRates', () => {
+    const mockedPricesService: AbstractTokenPricesService = {
```

Comment:
Nit: It looks like this mock is shared between tests. This is probably not hazardous right now because we don't modify this directly, we use `Jest` to modify and spy on it, and that gets reset between tests. But in general I'd recommend avoiding shared mocks altogether to reduce the changes that we'll accidentally introduce changes in one test that affect another, breaking test isolation.

We can do that by creating a function to return this mock, rather than using it directly (e.g. `createMockPriceService()`)

This pattern is explained further in our contributor docs here: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#reset-shared-variables

Reply:
Thanks I will update this 🙏